### DISCUSSION TITLE
Implement keyDown event

### DISCUSSION THREAD

**Discussion by isakemanuel** [keathley/wallaby#620] [Ideas]
In a Phoenix project I am working on, we have keyboard shortcuts, many of which are available outside of input fields and some globally in the app (F1/? to bring up help). With Wallaby, it seems `send_keys` can only be used while in an input field, but this means most of the keyboard shortcuts cannot be tested (some of them are not supposed to be accessible from within an input, to avoid clashing with regular typing). 

[In ChromeDriver and Selenium there is a keyDown event](https://www.selenium.dev/documentation/webdriver/keyboard/#keydown), which does let me test the keyboard shortcuts properly. With that event, you can use any element (e.g. the body tag). 

I think it would be useful to be able to send this event with Wallaby.

**Comment by michallepicki** [comment]
When you [pass a Session](https://github.com/elixir-wallaby/wallaby/blob/main/lib/wallaby/webdriver_client.ex#L516) and not an element, I think this is what should be happening? [Old selenium protocol docs](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidkeys)

**Reply by michallepicki** [reply]
Try using [Wallaby.Browser.send_keys/2](https://hexdocs.pm/wallaby/Wallaby.Browser.html#send_keys/2) with session (not input element) as first argument. Docs seem to be missing which could be improved...

**Reply by isakemanuel** [reply]
Doing so gives me the following error:
```
     ** (RuntimeError) sendKeysToActiveElement
     Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:32.194Z'
     System info: host: '***', ip: '***', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.16', java.version: '1.8.0_301'
     Driver info: driver.version: unknown
     code: |> Wallaby.Browser.send_keys(["?"])
     stacktrace:
       (wallaby 0.28.1) lib/wallaby/httpclient.ex:136: Wallaby.HTTPClient.check_for_response_errors/1
       (wallaby 0.28.1) lib/wallaby/httpclient.ex:56: Wallaby.HTTPClient.make_request/5
       (wallaby 0.28.1) lib/wallaby/webdriver_client.ex:518: Wallaby.WebdriverClient.send_keys/2
       (wallaby 0.28.1) lib/wallaby/browser.ex:513: Wallaby.Browser.send_keys/2
       ***/home_page_test.exs:17: (test)
```
